product_pricelist_alternative/models/product_pricelist_item.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    alternative_pricelist_policy = fields.Selection(
+        selection=[
+            ("use_lower_price", "Use lower price"),
+            ("ignore", "Ignore alternatives"),
+        ],
+        default="use_lower_price",
+        required=True,
+    )
+
+    def _compute_price(self, product, quantity, uom, date, currency):
+        price = super(PricelistItem, self)._compute_price(
+            product, quantity, uom, date, currency
+        )
+        if self.alternative_pricelist_policy == "ignore":
+            return price
+
+        if self.alternative_pricelist_policy == "use_lower_price":
```

Comment:
Doing a explicit check on the policy as we could imagine to have other  `alternative_pricelist_policy` values

Reply:
Q: as the order of the pricelists impacts the result: how do you ensure full control on which alternative pricelist will take precedence?

Reply:
In your example above you are returning the first alternative_price lower than original_price. But don't want that.
We want the minimum between original_price and alternative prices

Reply:
Correct, you can adapt that. I just wanted to show you a different approach that does not require using an IF every time even when not needed :wink: